### PR TITLE
[MDS-306] Align Dialog and Popover pages

### PR DESCRIPTION
**Pull request by tkullisaar**
Renamed Dialog page into Modal and updated url, also for Popover.
Made both pages visible on side menu.

### CODE REVIEW

**Review by margusp2heathmont — APPROVED** [review]

**Comment by margusp2heathmont** [comment]
Good job! Might cause issues in other projects, so notification required.

File: lib/moon_web/router.ex [hunk 1]
```diff
@@ -78,12 +78,12 @@ defmodule MoonWeb.Router do
         live("/components/date/single-date", MoonWeb.Pages.Components.Date.SingleDatePage)
         live("/components/date/range-date", MoonWeb.Pages.Components.Date.RangeDatePage)
 
-        live("/components/dialog/modal", MoonWeb.Pages.Components.Dialog.ModalPage)
+        live("/components/modal", MoonWeb.Pages.Components.ModalPage)
```
Comment (margusp2heathmont) [inline lib/moon_web/router.ex:81]: Should you update component itself?

**Review by margusp2heathmont — CHANGES_REQUESTED** [review]
Please see comments.

File: lib/moon_web/router.ex (hunk 1 above)
Comment (tkullisaar) [reply lib/moon_web/router.ex:81]: I didnt't change componets, changes only on moon_web directory.

Comment (margusp2heathmont) [reply lib/moon_web/router.ex:81]: Ok, but there is 1:1 mapping between documentation page and real component. You just can not change 1 without changing other.

Comment (phgrey) [reply lib/moon_web/router.ex:81]: but moving components themselves is a breaking change. any reason for this?

**Review by phgrey — APPROVED** [review]

File: lib/moon_web/router.ex (hunk 1 above)
Comment (margusp2heathmont) [reply lib/moon_web/router.ex:81]: Let's break it then. It is search and replace fix. Let's try to keep things nice and tidy.

**Review by margusp2heathmont — APPROVED** [review]

**Review by phgrey — APPROVED** [review]

**Pull request closed without merging**